addons/pos_restaurant/static/src/css/restaurant.css

```diff
@@ -315,3 +315,80 @@
     margin-right: 4px;
 }
 
+/* ------ SPLIT BILL SCREEN ------- */
+
+/* ------ WIDER SCREEN ------ */
+@media screen and (min-width: 768px) {
+    .splitbill-screen.screen .main {
+        flex-flow: row nowrap;
+    }
+    .splitbill-screen.screen .main .lines {
+        border-right: dashed 1px rgb(215,215,215);
+    }
+    .splitbill-screen.screen .main .controls {
+        flex: 1;
+    }
+}
+/* ------ NARROWER SCREEN ------ */
+@media screen and (max-width: 768px) {
+    .splitbill-screen.screen .main {
+        flex-flow: column nowrap;
+    }
+    .splitbill-screen.screen .main .controls {
+        border-top: dashed 1px rgb(215,215,215);
+    }
+}
+.splitbill-screen.screen .contents {
+    display: flex;
+    flex-flow: column nowrap;
+    margin: 0px auto;
+    max-width: 1024px;
+    border-left: dashed 1px rgb(215,215,215);
+    border-right: dashed 1px rgb(215,215,215);
+    height: 100%;
+}
+.splitbill-screen.screen .contents .main {
```

Comment:
seems that the rules for ".splitbill-screen.screen .contents .main " and ".splitbill-screen.screen .main" are the same element, maybe check if there is not redundancy, and porbably you had to specify some due to previous comment

Reply:
I'll check this redundancy.

Reply:
There is no redundancy. I'm setting .main in the media queries to replace the flex-flow based on the size of the screen. I'll add .content to the selector in the media queries to be more explicitly.